.tools/readmes/render.py

```diff
@@ -331,11 +332,13 @@ def write(self):
                 self.readme_filename,
                 f'{self.lang_config["service_folder"]}/{config.saved_readme}',
             )
-        os.remove(self.readme_filename) #  Do this so that new files are always updated to the correct case (README.md).
```

Comment:
`Path(self.readme_filename).unlink(missing_ok=False)`

Reply:
It's a draft.

Reply:
So what. It seems overly pedantic. I think the former is more readable.

Reply:
I know we haven't done that everywhere, but we have moved to preferring Path in -tools; given that this is going into a Path, it seems reasonable to stick with the Path method rather than going back to `os`.